src/figuresClasses.ts

```diff
@@ -1,19 +1,94 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
 export class Triangle {
+  shape: string;
```

Comment:
```suggestion
  public shape: Shape;
```
You can use typescript features here and the same for other arguments